http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -349,13 +370,16 @@ END
 # Create a network that allows containers to talk to each other via exposed ports
 docker network create -d bridge app_network
 
+# Docker compose config do dump the env variable to the yaml
```

Comment:
I think this is really just for logging/debugging, it dumps the rendered yaml to the init script log.